enterprise/server/remote_execution/platform/platform.go

```diff
@@ -226,6 +228,15 @@ func ParseProperties(task *repb.ExecutionTask) *Properties {
 	// Only Enable VFS if it is also enabled via flags
 	vfsEnabled := boolProp(m, enableVFSPropertyName, false) && *enableVFS
 
+	envOverrides := stringListProp(m, EnvOverridesPropertyName)
+	for _, prop := range stringListProp(m, EnvOverridesBase64PropertyName) {
+		b, err := base64.StdEncoding.DecodeString(prop)
+		// Ignore decode errors
```

Comment:
probably shouldn't ignore these -- makes it hard for a user to tell why their override isn't working

Reply:
`ParseProperties` returns `*Properties` instead of `*Properties, error` (my mistake) and there are a lot of callsites, so I think that would result in this PR being pretty big - we might want to do that separately

Reply:
I could send a separate PR to change the return and return a `status.FailedPreconditionError` when the base64 decode fails.